ambry-router/src/test/java/com.github.ambry.router/MockServer.java

```diff
@@ -64,7 +63,7 @@
   private boolean getErrorOnDataBlobOnly = false;
   private final ClusterMap clusterMap;
   private final String dataCenter;
-  private final HashMap<RequestOrResponseType, Integer> requestCounts = new HashMap<>();
+  private final ConcurrentHashMap<RequestOrResponseType, Integer> requestCounts = new ConcurrentHashMap<>();
```

Comment:
I see what you mean. Fixed this using `LongAdder` (supposedly better than `AtomicLong` for this purpose - not that it matters much here).